mite/logoutput.py

```diff
@@ -96,14 +96,18 @@ def process_message(self, message):
             del self._resp_time_recent[:]
             self._req_recent = 0
             self._error_recent = 0
-        if msg_type == 'http_metrics':
-            self._resp_time_recent.append(message['total_time'])
+        if msg_type == "http_metrics":
+            self._resp_time_recent.append(message["total_time"])
             self._req_total += 1
             self._req_recent += 1
-        elif msg_type == 'http_selenium_metrics':
-            self._resp_time_recent.append(message['total_time'])
+        elif msg_type == "http_selenium_page_load_metrics":
+            self._resp_time_recent.append(message["total_time"])
             self._req_total += 1
             self._req_recent += 1
-        elif msg_type in ('error', 'exception'):
+        elif msg_type == "http_selenium_network_resource_metrics":
```

Comment:
`if msg_type in {"http_metrics", ...}`

Reply:
Not sure why I didn't clock that, updated this